packages/shared/src/components/widgets/PostToc.tsx

```diff
@@ -52,12 +53,12 @@ export default function PostToc({
   );
 
   const titleClass =
-    'flex items-center py-3 px-4 typo-body text-theme-label-tertiary';
-  if (collapsible) {
+    'flex items-center py-3 px-4 typo-body text-theme-label-tertiary border border-theme-divider-quaternary';
+  if (!collapsible) {
```

Comment:
Was this intended or for testing purpose?

Reply:
Good spot, was for the screenshot. Removed now

Reply:
Feel free to merge :)